app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -64,10 +64,14 @@ def send_fast_track_engineer_email_for_testing(form526_submission_id, error_mess
     end
 
     def get_icn(form526_submission)
-      account = Account.where(idme_uuid: form526_submission.user_uuid).first
-      account = Account.where(logingov_uuid: form526_submission.user_uuid).first if account.blank?
-      account = Account.where(edipi: form526_submission.auth_headers['va_eauth_dodedipnid']).first if account.blank?
-      account.icn if account.present? && account.icn.present?
+      account(form526_submission).icn.presence
+    end
+
+    def account(form526_submission)
+      user_uuid = form526_submission.user_uuid
+      @account ||= Account.where(idme_uuid: user_uuid)
+                          .or(Account.where(logingov_uuid: user_uuid))
+                          .or(Account.where(edipi: form526_submission.auth_headers['va_eauthdodedipnid'])).first!
```

Comment:
I was a little nervous about the use of `or` here because of the assignment but checking it in the console it seems solid. 🙌

```
irb(main):001:0> bird = false or true
=> true
irb(main):002:0> bird
=> false
irb(main):003:0> bird = false || true
=> true
irb(main):004:0> bird
=> true
irb(main):005:0> bird = nil || true
=> true
irb(main):006:0> bird
=> true
irb(main):007:0> bird = nil or true
=> true
```

Reply:
<img src="https://media4.giphy.com/media/zlVf2eSgXIFFuTnEhz/giphy.gif"/>